src/Microsoft.Identity.Client/Cache/StorageJsonKeys.cs

```diff
@@ -50,5 +50,12 @@ internal static class StorageJsonKeys
         public const string ExtendedExpiresOn = "extended_expires_on";
         public const string ClientInfo = "client_info";
         public const string FamilyId = "family_id";
+
+        // todo(cache): this needs to be added to the spec.  needed for OBO flow on .NET.
+        public const string UserAssertionHash = "user_assertion_hash";
+
+        // previous versions of msal used "ext_expires_on" instead of the correct "extended_expires_on".
+        // this is here for back compat
+        public const string ExtendedExpiresOn_MsalCompat = "ext_expires_on";
     }
```

Comment:
Did you validate that all these names follows the schema? And that those aligns with Python?

Reply:
Yes.  The only (known) mismatch between us and python is that python is storing the scopes ("target") as an array of strings, and the spec says it should be a space-delimited string.  so i've disabled the python validation test until Abhi/Ray get me a new .bin file (I talked with Abhi yesterday and they're working on it).

Reply:
Done at [this PR](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/19)